### PR TITLE
update to reflect Cloud-only or Federated identity model

### PR DESCRIPTION
**Pull request by VLG17**
addressed both issues:
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2936 
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2351

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by jdeckerms** [comment]
I have to get these changes confirmed by the product group before they can be merged.

**Comment by mypil** [comment]
@TokyoScarab - can you please do the copy editor review for this PR? Thanks.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -43,14 +43,20 @@ From its release, Windows 10 has supported remote connections to PCs that are jo
...
+  >
   >In Windows 10, version 1709, the user does not have to sign in to the remote device first.
   >
   >In Windows 10, version 1709, you can add other Azure AD users to the **Administrators** group on a device in **Settings** and restrict remote credentials to **Administrators**. If there is a problem connecting remotely, make sure that both devices are joined to Azure AD and that TPM is functioning properly on both devices.
 
   4. Enter **Authenticated Users**, then click **Check Names**. If the **Name Not Found** window opens, click **Locations** and select this PC.
 
   >[!TIP]
-  >When you connect to the remote PC, enter your account name in this format: `AzureADName\YourAccountName`.
+  >When you connect to the remote PC, enter your account name in this format: `AzureAD\<AzureAD UPN>`. In some Windows builds the *AzureAD* prefix is no longer required. You can just use your *AzureAD UPN*.
```
Comment (TokyoScarab) [inline windows/client-management/connect-to-remote-aadj-pc.md]:
```suggestion
  >When you connect to the remote PC, enter your account name in this format: `AzureAD\<AzureAD UPN>`. In some Windows builds, the *AzureAD* prefix is no longer required. You can just use your *AzureAD UPN*.
```

**Comment by mypil** [comment]
@VLG17 - can you please merge the new commits on the copy/edit review done by Tokyoscarab? 

Thank you.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ From its release, Windows 10 has supported remote connections to PCs that are jo
   4. Enter **Authenticated Users**, then click **Check Names**. If the **Name Not Found** window opens, click **Locations** and select this PC.
 
   >[!TIP]
-  >When you connect to the remote PC, enter your account name in this format: `AzureAD\<AzureAD UPN>`. In some Windows builds the *AzureAD* prefix is no longer required. You can just use your *AzureAD UPN*.
+  >When you connect to the remote PC, enter your account name in this format: `AzureAD\<AzureAD UPN>`. In some Windows builds, the *AzureAD* prefix is no longer required. You can just use your *AzureAD UPN*.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md]: When connecting to the remote PC, enter your account name in this format: `AzureAD\<AzureAD UPN>`. In some Windows builds the *AzureAD* prefix is not required and you only need to use your *AzureAD UPN*.

**Comment by jdeckerms** [comment]
Still pending product group confirmation

**Comment by mypil** [comment]
@jdeckerms - Do you have any updates from the product group? Thanks

**Comment by jdeckerms** [comment]
@mypil I will post when I have an answer, thanks.

**Comment by jdeckerms** [comment]
@VLG17 I made changes to your pull request per the product group.

**Pull request merged**